docs/tau-updates.md

```diff
@@ -0,0 +1,128 @@
+})
+```
+
+2. In `src/AVAILABLE_MODULES`, You'll need to import your invulnerablity updates:
+
+```ts
+// with all the imports
+import {identifier} from './bosses/identifier'
+
+// In the BOSSES section of AVAILABLE_MODULES
+const AVAILABLE_MODULES: AvailableModules = {
+	// ...
+	BOSSES: {
+		[BOSSES.IDENTIFIER.logId]: identifier,
+	},
+}
+```
+
+# Setting up your config
+
+In the `Invulnerability.ts` file for your fight, there are 3 options per actor ID:
+
+
+| Param | Option | Description |
+| ----- | ---------: | ----------- |
+| start | 'tau'      | Use targetability update from fflogs (the default) |
+|       | 'firstTap' | Use the first registered hit on the enemy as a synthetic targetability update |
+| end   | 'tau'   | use targetability update from fflogs (the default) |
+|       | 'overkill' | Use the last hit registered on a boss (that causes overkill) as a synthetic targetability update |
+| mirrorToInvuln | boolean | "mirrors" untargetable moments with invulnerable moments. Set to 'false' for enemies that can be untargetable but still take damage (dot tickets, other effects, etc). Defaults to true. |
+| exclude | boolean | exclude the target from the fight in terms of targeting (default is false) |
+
+For example, let's say there are 3 actors (4567, 4568, and 4569) with different properties:
+
+* The first actor (id 4567) gets no targetability or death updates, but is present throughout the entire fight. This means you need a 'start' and 'end' parameter to clearly define the target's availability
+* The second actor (id 4568) doesn't have parameters indicating that it is available to hit; however, it has death events
+* The third actor (id 4569) just fires lasers, isn't targetable, but generally makes your analysis miserable.
+
+There might be other actors, but we receive correct information from fflogs, so those aren't an issue, and shouldn't be provided in the configuration.
+
+With that in mind, you could update the target's invulnerability windows in `Invulnerability.ts`:
+
+```ts
+
+import {
+	Invulnerability as CoreInvulnerability,
+	ActorsConfig,
+} from 'parser/core/modules/Invulnerability'
+
+export class Invulnerability extends CoreInvulnerability {
+	protected actorConfig: ActorsConfig = {
+		// Target 1 - no TAU or death events
+		4567: {start: 'firstTap', end: 'overkill'},
+		// Target 2 - no TAU, but provides death event(s)
+		4568: {start: 'firstTap'},
+		// Target 3 - can't target them
+		4569: {exclude: true},
+	}
+}
+```
+
+# Testing
+
+(Temporarily) modify `parser/core/modules/invulnerability.tsx` and set `static debug` to `true`. With your updates in place (or while tweaking your updates), the timeline should have an additional section indicating target(s) in various states. If the target doesn't show up and it needs to be there, add the actor ID and configure as appropriate. If the actor is already there but it's lingering or not showing up at the right time, also tweak these values.
```

Comment:
might be worth mentioning the debug mode earlier in the doc - people will usually want to check that first, then set up overrides

---

as with log id, probably should outline how to get actor ids - the debug mode does output ones it's aware of, but as with the discussion in dev-general the other day, isn't always all-inclusive.